cloud/endagaweb/templates/dashboard/tower_detail/deregister.html

```diff
@@ -98,25 +121,50 @@ <h4 class='modal-title'>
       </a>
     </p>
   </div>
+<!-- removed  check of Tower  status == 'No Data' -->
+  {% if endaga_version > '00000.00003.00019' %}
```

Comment:
can you add the check back please? I think it's simpler if users can remove unregistered towers without the extra check box.